abstract.hbs

```diff
@@ -5,7 +5,12 @@
   List</a>.
   </p>
   <p>
-    In addition to the test results below,
+    The
+    <a href="https://github.com/w3c/vc-data-model-2.0-test-suite?tab=readme-ov-file#test-suite-http-api">test suite API</a>
+    requires raw JSON responses such that implementations
+    using VC JOSE/COSE or other enveloped response bodies are required to unwrap
+    the envelopes prior to responsing. Consequently, in addition to the test
+    results below,
     <a href="https://github.com/w3c/vc-bitstring-status-list-test-suite/issues/61">Tradeverifyd reports</a>
     a successful implemention of every feature, including the ability to
     host multiple statuses in a single list.
```

Comment:
```suggestion
    the envelopes prior to responding. Consequently, in addition to the test
    outputs reproduced below,
    <a href="https://github.com/w3c/vc-bitstring-status-list-test-suite/issues/61">Tradeverifyd reports</a>
    a successful implementation of every feature, including the ability to
    host multiple statuses in a single list.
```

Reply:
Thanks @TallTed! What do you think about "in addition to the test outputs produced below" (since they aren't "reproduced")?

Reply:
They're "reproduced" because they're not run upon GET of that page.

But that "reproduced" could be changed to "shown" or "found" or various other words.

Reply:
@TallTed can we just go back to "test results"?

Reply:
Done.